src/common/styles/componentsStyles.ts

```diff
@@ -70,12 +79,21 @@ export const Input = styled.input`
   width: 14rem;
   height: 2rem;
   padding: 0.5rem;
+
+  @media (max-width: 540px) {
+    padding: 0.5rem 0;
+  } ;
 `;
 
 export const Select = styled.select`
   font-family: Rubik, sans-serif;
-  padding: 0.5rem 1rem;
-  border-radius: 10px; ;
+  font-size: 1.2rem;
+  padding: 0.1rem 0.2rem;
+  border-radius: 10px;
+
+  @media (max-width: 710px) {
```

Comment:
it's better when all the media queries are consistent and coming from your theme